lib/Twig/Extension/Core.php

```diff
@@ -1402,6 +1402,12 @@ function twig_include(Twig_Environment $env, $context, $template, $variables = a
 
             throw $e;
         }
+    } catch (\Exception $e) {
```

Comment:
Does PHP 5 choke on `catch (Throwable $e)`? 

Should that be before or after catching Exception? It probably doesn't matter in this case. I just remember reading somewhere that to make code compatible with PHP 5 `Throwable` should be caught first...